### ISSUE TITLE
[UI] new task/group viewer reloads entire graph when going from group -> task -> group

### ISSUE THREAD

**Issue by mozbhearsum** [taskcluster/taskcluster#665]
**Describe the bug**
When using the new task/group viewer, the group is unexpectedly reloaded when I start from a group, view a task, and then go back to the group. On large task groups, this ends up being a significant hit, and is a regression from the old UI where the group would remain fully loaded.

**To Reproduce**
1. Go to https://taskcluster-web.netlify.com/tasks/groups/Jw3OURbYTN2zLIuBKzPVDA
2. Click on any task
3. Click on "Task Group"

**Expected behavior**
The previously loaded group should be there, not be reloaded from scratch.

**Platform (please complete the following information):**
 - OS: Linux
 - Firefox
 - 68.0

**Comment by helfi92** [comment]
Agreed. We should have this fixed. In the meantime, feel free to open tasks in a new tab. Sorry for the trouble.

**Comment by projectyang** [comment]
Hey @helfi92, I'd like to help with this.

Based on the docs, it seems like there's a requirement for network-only on views like this. Would changing the fetch policy to 'cache-and-network' be a suitable compromise?

**Comment by helfi92** [comment]
That may or may not work. I'm not sure how this would interact with the subscriptions that we have in https://github.com/taskcluster/taskcluster/blob/master/ui/src/views/Tasks/TaskGroup/index.jsx#L242-L248. When updates come (e.g., a task status has changed), we update the cache using [`subscribeToMore`](https://www.apollographql.com/docs/react/advanced/subscriptions/#subscribetomore) https://github.com/taskcluster/taskcluster/blob/bcb6565eb437b17c315cfb73cd68e02952d5b953/ui/src/views/Tasks/TaskGroup/index.jsx#L238.

You can test this by having the production version of the site running on a separate tab/window and then staying on the task view until you see changes in the task state progress of the production site. Once you see changes, you can try navigating to the group view and seeing if the state numbers are the same.

I'm curious to know if this will actually fix this issue. It will make so many people happy :)
